servers/rendering/shader_compiler.cpp

```diff
@@ -127,6 +127,8 @@ static String _interpstr(SL::DataInterpolation p_interp) {
 	switch (p_interp) {
 		case SL::INTERPOLATION_FLAT:
 			return "flat ";
+		case SL::INTERPOLATION_NOPERSPECTIVE:
+			return !RS::get_singleton()->is_low_end() ? "noperspective " : "";
```

Comment:
Note: this disables it on the gl_compatibility renderer entirely, even when supported.

Reply:
I think this would still be fine though, if there's proper documentation on when the feature will fail then I think that's acceptable for now. I would rather have this feature that some people can use rather than not have it at all.